### DISCUSSION TITLE
Tuya Homebridge Plugin v1.5.0 is released!

### DISCUSSION THREAD

**Discussion by Hanh94** [tuya/tuya-homebridge#146] [Announcements]
## Important Note

Please update to V1.5.0 because previous versions had cross-region access security issues with the plugin's login.

## Fixed Issues

- how to change endpoint from us to eu. #144
- unable to add devices to HomeKit getting error 2406 #143
- Constantly getting the 2406 error code when setting it up #142
- Can not login through tuya-homebridge and IFTTT #131
- Homebridge connect error #109
- Trouble 1013 #95

**Comment by t829913** [comment]
@Hanh94 has development stopped on this plugin? Last release was 9 months ago, no pull requests being authorised... seems like this has been abandoned?

**Reply by squishycat92** [reply]
See issue #242. It seems that all new issues are simply getting marked as stale and eventually getting closed. Hopefully it hasn't been abandoned, but judging from the lack of commits and comments by members of this repository, it seems that it isn't being developed (at least for now).

Unfortunately, it doesn't seem like we (the users) have much choice. iRayanKhan's [homebridge-tuya](https://github.com/iRayanKhan/homebridge-tuya/wiki) plugin hasn't had an update since August 30, 2021, while milo526's [homebridge-tuya-web](https://github.com/milo526/homebridge-tuya-web) is essentially an unofficial version of this plugin that has support for less devices.

Hopefully, the development team can come back soon and restart development on this plugin, particularly on many features that are important to the Homebridge community (LAN control, Adaptive Lighting, etc.).

**Reply by t829913** [reply]
I'm wondering if Tuya are cooling their jets on this and focussing on support for Matter / Thread instead... they've announced that they intend to support it and I suppose this might make the homebridge plugin redundant?

**Reply by squishycat92** [reply]
Doubt it. Matter, Thread, and native HomeKit support both require more system resources - something that a small IoT device like a smart plug probably wouldn't have. Besides, with the large amount of devices that Tuya supports, I find it hard to believe that *every* device will be able to get its own Matter, Thread, and/or native HomeKit update.
